pommerman/network/ion_client/__init__.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python
+"""IonClient Entry Point  
+
+This library allows playing of matches on network via the WebSockets protocol.  
+Functions:  
+init() - If you want to run the application normally  
+intent(network) - If you want to pass a pre-existing network object but want to
+prompt the user about what they want to do  
+match(network, room=False, import_name=False, agent_class=False) - If you want 
+to start a match directly  
+replay(network, id=False) - If you want to start a replay directly  
+"""
+
+import ui
+from . import constants
+# This is for switching between languages incase there are more
+Strings = constants.LangEN
```

Comment:
why change the name here? why not just leave it as constants.LangEN in the module?

Reply:
` # This is for switching between languages incase there are more ` The reason was right above that

Reply:
There aren't more languages and Strings is a poor name for this. Please just use constants.LangEN or change LangEN to something you prefer.

Reply:
I suppose I agree.